test/test_time_series.jl

```diff
@@ -575,405 +582,6 @@ end
     )
 end
 
-function validate_generated_initial_times(
```

Comment:
Do we still want to have some of the functionalities from these tests?

Reply:
I'm pretty sure this is all pointless now